tests/functional/jaws/src/test/java/com/scality/StreamV4AuthTest.java

```diff
@@ -0,0 +1,140 @@
+package com.scality;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.FileInputStream;
+import java.util.Arrays;
+import java.util.Collection;
+import java.io.RandomAccessFile;
+
+import com.amazonaws.SDKGlobalConfiguration;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.BeforeClass;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.runners.Parameterized;
+import org.junit.runner.RunWith;
+import java.io.File;
+import java.io.FileReader;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.JSONObject;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.S3ClientOptions;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.S3Object;
+
+
+@RunWith(Parameterized.class)
+public class StreamV4AuthTest {
+    private Integer fileSize;
+
+    protected static String accessKey;
+    public static  String getAccessKey() { return accessKey; }
+    protected static String secretKey;
+    public static String getSecretKey() { return secretKey; }
+    protected static String transport;
+    public static String getTransport() { return transport; }
+    protected static String ipAddress;
+    public static String getIpAddress() { return ipAddress; }
+    protected static AmazonS3 s3client;
+    public static AmazonS3 getS3Client() { return s3client; }
+    public static final String bucketName = "streambucket";
+    public static final String objName = "streamObject";
+    public static final String fileName = "obj.txt";
+
+    @BeforeClass
+    public static void initialize() throws Exception {
+        System.setProperty(SDKGlobalConfiguration.DISABLE_CERT_CHECKING_SYSTEM_PROPERTY, "true");
+        JSONParser parser = new JSONParser();
+        String path = Paths.get("../config.json").toAbsolutePath().toString();
+        System.out.println(path);
```

Comment:
perhaps this System.out.println should be remove.
Or the message could be improve to make it clear that this is the path of the configuration file being used.
This is not a blocker in my opinion however.